### PR TITLE
Updated Webhook Builders

### PR DESCRIPTION
**Pull request by AustinHay**
@aeromusek @Johnjoe220 

- Updated the following params:
sub_campaign={{click.link_data.~campaign}}&
sub_keyword={{click.link_data.~tags}}&
tracking_id={{click.branch_id}}&
device_ip={{click.browser.metadata.ip}}&
user_agent={{click.browser.metadata.userAgent}}
sub_placement={{click.link_data.~channel}}

- Updated styling with some form-control

- cleaned up some formatting for Ajust and Tune.

### CODE REVIEW

**Comment by AustinHay** [comment]
cc @Sarkar to make sure those params are right.

A

File: _data/sidebar.yml [hunk 1]
```diff
@@ -57,6 +57,13 @@
         - appboy
     - title: Measurement Platforms
       children:
+<<<<<<< HEAD
```
Comment (aeromusek) [inline _data/sidebar.yml]: @AustinHay looks like the merge went funky here...would you like me to clean it up? If you don't want these in the sidebar, feel free to just take them out — probably no advantage to commenting :+1:

**Comment by Sarkar** [comment]
tags look good to me

**Comment by Johnjoe220** [comment]
@lilabjorg do we need your final approval to merge this?  If so, can we go ahead and do that so we can finish QA on Tune?

**Comment by lilabjorg** [comment]
@Johnjoe220 I'll pull it down and take a look - make sure there are no weird conflicts

**Pull request merged**